pkg/plugins/plugins.go

```diff
@@ -73,6 +73,12 @@ type Plugin struct {
 	activateWait *sync.Cond
 }
 
+// PathsRelativeTo returns the path to which all paths returned by the plugin are relative to.
+// For legacy plugins, this is always relative to the host's root directory.
+func (p *Plugin) PathsRelativeTo() string {
```

Comment:
I realize this name sucks, so alternatives are welcome. Rootfs? Basepath? Hostpath?